src/styles/utils/_function.scss

```diff
@@ -0,0 +1,3 @@
+@function calendarWidth ($calendar-size, $wrapper-gap, $wrapper-side-pading) {
+  @return 7 * $calendar-size + 2 * $wrapper-side-pading + 7 * $wrapper-gap;
```

Comment:
Get the number as an argument. 
Create a meaningful name for it.

Reply:
What number do you mean here?

Reply:
7 - it's an argument. You can call it f.e. `$cols-count`